app/src/src/stylesheets/views/views.jsx

```diff
@@ -0,0 +1,48 @@
+
+/* Used in AdView, EventView, ListingView, PollView */
+import { StyleSheet } from "react-native";
+
+const commonViewStyles = StyleSheet.create({
```

Comment:
We can add 'export' infront of const here to save lines.
`export const commonViewStyles = StyleSheet.create({`

Also, we should rename this constant to something like 'sharedItemViewStyles'